ImplementationGuide/markdown/Organisation/Organisation_Motivation.md

```diff
@@ -0,0 +1,11 @@
+### Motivation
+
+Die Abbildung der Aufbauorganisation eines Krankenhauses dient der Festlegung von Zuständigkeiten und (Entscheidungs-)Verantwortungen von Organisationseinheiten (z.B. Stationen, Fachabteilungen und -bereichen,) in strukturierter Form. 
```

Comment:
Generell sehe ich den Vorteil in der Abbildung einer Organization-Ressource darin, dass man die Hierarchie von Abteilungen, Kliniken, etc. abbilden kann (per Organization.partOf). Falls es nur darum geht die Zuständigkeit zu anmontieren, könnte dies auch "inline" per Display-Value (Name der Organization) und Identifier (z.B. IKNR der Organization) erfolgen.